qemu/tests/cfg/balloon_in_use.cfg

```diff
@@ -0,0 +1,30 @@
+- balloon_in_use:
+    type = driver_in_use
+    only Windows
+    extra_params += "-device virtio-balloon-pci,id=ballooning"
+    iterations = 5
+    ratio = 0.5
+    login_timeout = 800
+    start_vm = yes
+    kill_vm_on_error = yes
+    driver_name = "balloon"
+    run_bgstress = balloon_check
```

Comment:
balloon_check is not the same as the stress from the requirement, is it acceptable?